### ISSUE TITLE
Something is broken in 0.65.0 version.

### ISSUE THREAD

**Issue by YevhenKap** [fannheyward/coc-rust-analyzer#1006]
Hi! First of all, thanks for the great extension! I am using it for a while and find it pretty good, but after updating the extension to `0.65.0` version it stops working(

**What's the output of `:CocInfo`**

```
## versions

vim version: VIM - Vi IMproved 9.0 9000000
node version: v16.15.1
coc.nvim version: 0.0.81-2522eee5 2022-06-14 19:03:29 +0800
coc.nvim directory: /Home/.vim/pack/coc/start/coc.nvim
term: Apple_Terminal
platform: darwin

# ...

2022-06-30T23:00:01.913 ERROR (pid:17502) [extensions] - Error on createExtension coc-rust-analyzer from /Home/.config/coc/extensions/node_modules/coc-rust-analyzer/lib/index.js Error: ENOENT: no such file or directory, open '/Home/.config/coc/extensions/node_modules/coc-rust-analyzer/lib/bridge.js'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at node_modules/vm2/lib/vm.js (/Home/.config/coc/extensions/node_modules/coc-rust-analyzer/lib/index.js:42286:123)
    at __require (/Home/.config/coc/extensions/node_modules/coc-rust-analyzer/lib/index.js:25:50)
    at node_modules/vm2/lib/main.js (/Home/.config/coc/extensions/node_modules/coc-rust-analyzer/lib/index.js:43703:9)
    at __require (/Home/.config/coc/extensions/node_modules/coc-rust-analyzer/lib/index.js:25:50)
    at node_modules/vm2/index.js (/Home/.config/coc/extensions/node_modules/coc-rust-analyzer/lib/index.js:43719:23)
    at __require (/Home/.config/coc/extensions/node_modules/coc-rust-analyzer/lib/index.js:25:50)
    at node_modules/degenerator/dist/src/index.js (/Home/.config/coc/extensions/node_modules/coc-rust-analyzer/lib/index.js:43731:17)
    at __require (/Home/.config/coc/extensions/node_modules/coc-rust-analyzer/lib/index.js:25:50) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/Home/.config/coc/extensions/node_modules/coc-rust-analyzer/lib/bridge.js'
}
```

**What's the output of `:CocCommand rust-analyzer.serverVersion`**

```
[coc.nvim] Error on notification "runCommand": Command: rust-analyser.serverVersion not found.
```

I have updated the Rust to `1.62.0` version, though it may not be the problem, but I think I should mention it just in case.
Also, I have tried to reinstall the extension, but it didn't work(

**Comment by fannheyward** [comment]
Fixed in v0.65.1, sorry for the trouble.

**Comment by YevhenKap** [comment]
It's okay, thank you)
